core/ide/src/main/java/org/alice/stageide/oneshot/OneShotUtilities.java

```diff
@@ -72,16 +75,23 @@ public static List<StandardMenuItemPrepModel> createMenuItemPrepModels(InstanceF
       UserField field = thisFieldAccessFactory.getField();
       models.add(RenameFieldComposite.getInstance(field).getLaunchOperation().getMenuItemPrepModel());
       //Prevent users from deleting the camera or the scene
-      if (field.getValueType().isAssignableTo(SCamera.class) || field.getValueType().isAssignableTo(SScene.class)) {
-        //pass
-      } else {
+      if (!field.getValueType().isAssignableTo(SCamera.class) && !field.getValueType().isAssignableTo(SScene.class)) {
         models.add(DeleteFieldOperation.getInstance(field).getMenuItemPrepModel());
       }
       //      if( field.getValueType().isAssignableTo( SBiped.class ) && ( field.getValueType() instanceof NamedUserType ) ) {
       //        models.add( new AnimatorInputDialogComposite( (NamedUserType)field.getValueType() ).getOperation().getMenuItemPrepModel() );
       //        models.add( new PoserInputDialogComposite( (NamedUserType)field.getValueType() ).getOperation().getMenuItemPrepModel() );
       //      }
       models.add(RevertFieldOperation.getInstance(field).getMenuItemPrepModel());
+
+      if (!StorytellingSceneEditor.getInstance().isStartingCameraView() ) {
+        if (!field.getValueType().isAssignableTo(SCamera.class) &&
```

Comment:
Alice checkstyle is grumpy about these lines.
Feel free to curse my name as you change them.